### PR TITLE
Updated work on sig_tracker 

### PR DESCRIPTION
**Pull request by sharifa2708**
This pull request is a part of work required to make BITS-ACM backend team work to design models , make a basic user interface and to create necessary view functions. To achieve this following changes have been done:

1. Added models, views and templates(index.html and detail.html) to "about_sig" app. Updated the urls.py to map urls with views. This app functions to provide the basic templates for home page and provides a link to progress of individual sigs . (Basic template may be later changed by modifying index.html and detail.html)

2. Added a navigation bar, but the buttons have not yet been url mapped.

3. Added models to "dev_sig" app.(This app is for the progress of development sig. Feedback mechanism is yet to added.)

*Also a generic view is to be added*

### CODE REVIEW

**Comment by twitu** [comment]
remove all compiled files, ignored files using git rm --cached. this includes pycache, migrations and .idea/

File: README.md [hunk 1]
```diff
@@ -1,2 +1,55 @@
...
+## How to run?
+
+#### Setting up the Development Environment
+
+1.	For people using python3 virtual environment:
+
+To install python3 virtual environment, refer this-
+
+i.	
+
```
Comment (apb7) [inline README.md]: Remove all blank extra lines in this file.

File: sig_tracker/about_sig/admin.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+from django.contrib import admin
+from . models import Sig
+from . models import Topics
+
+admin.site.register(Sig)
+admin.site.register(Topics)
```
Comment (apb7) [inline sig_tracker/about_sig/admin.py]: Always add a newline to the end of the file.

File: sig_tracker/about_sig/models.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    topic_name = models.CharField(max_length = 1000)
+    co_ordinator = models.CharField(max_length=250)
+    conductors = models.TextField(blank=True)
+    date = models.DateTimeField(auto_now_add=True)
+    topic_gist = models.TextField(blank=True)
+    resources = models.TextField(blank=True)
+
+    def  __str__(self):
+        return self.topic_name
+
```
Comment (apb7) [inline sig_tracker/about_sig/models.py]: Remove these extra lines. Just keep a single newline at the end of the file.

File: sig_tracker/about_sig/templates/about_sig/detail.html [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+<h3>{{sig.co_ordinator}}</h3>
+
+<ul>
+    {%for sig in sig.topic_set.all %}
+
+        <li> {{topic.topic_name}}-{{topic.date}} </li>
+
+    {% endfor %}
+</ul>
```
Comment (apb7) [inline sig_tracker/about_sig/templates/about_sig/detail.html]: Newline?

File: sig_tracker/about_sig/templates/about_sig/index.html [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+           <a href=""> <h3>COMPETITIVE CODING </h3></a>
+    </div>
+
+  </div>
+</div><br>
+
+<footer class="container-fluid text-center">
+  <p>Contact</p>
+</footer>
+
```
Comment (apb7) [inline sig_tracker/about_sig/templates/about_sig/index.html]: Remove extra lines. Keep a single newline.

File: sig_tracker/about_sig/urls.py [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+
+app_name = 'about_sig'
+urlpatterns = [
+    #/about_sig/
+    url(r'', views.index, name='index'),
+    #/about_sig/1/
+    url(r'(?P<sig_id>[0-9]+)/$', views.detail, name='detail'),
+
+
+]
```
Comment (apb7) [inline sig_tracker/about_sig/urls.py]: Newline?

File: sig_tracker/about_sig/views.py [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+
+
+def detail(request, sig_id):
+    try:
+        sig = Sig.objects.get(pk=sig_id)
+    except Sig.DoesNotExist:
+        raise Http404("SIG does not exist")
+    return render(request, 'about_sig/detail.html', {'sig': sig})
+
```
Comment (apb7) [inline sig_tracker/about_sig/views.py]: Single new line.

File: sig_tracker/dev_sig/admin.py [hunk 8]
```diff
@@ -0,0 +1,4 @@
+from django.contrib import admin
+from . models import dev
+
+admin.site.register(dev)
```
Comment (apb7) [inline sig_tracker/dev_sig/admin.py]: New line?

File: sig_tracker/dev_sig/views.py [hunk 9]
```diff
@@ -0,0 +1,9 @@
+from django.shortcuts import render
+from django.http import HttpResponse
+
+def index(request):
+    return HttpResponse('''<h1> List of Topics under Development Sig</h1>''')
+
+def detail(request, dev_id):
+    return HttpResponse("<h2> Details for dev_id:" + str(dev_id) + "</h2>")
+
```
Comment (apb7) [inline sig_tracker/dev_sig/views.py]: Single newline only.

File: sig_tracker/dev_sig/urls.py [hunk 10]
```diff
@@ -0,0 +1,14 @@
...
+
+app_name = 'dev_sig'
+urlpatterns = [
+    #/dev_sig/
+    url(r'', views.index, name='index'),
+    #/dev_sig/1/
+    url(r'(?P<sig_id>[0-9]+)/$', views.detail, name='detail'),
+
+
+]
```
Comment (apb7) [inline sig_tracker/dev_sig/urls.py]: Newline?

File: sig_tracker/dev_sig/models.py [hunk 11]
```diff
@@ -0,0 +1,11 @@
...
+
+class dev(models.Model):
+    topic = models.CharField(max_length=2000)
+    summary = models.TextField(blank =True)
+    contact = models.TextField(blank=True)
+    date = models.DateTimeField(auto_now_add =True)
+# include feedback
+
+    def __str__(self):
+        return self.topic
```
Comment (apb7) [inline sig_tracker/dev_sig/models.py]: Newline?

**Pull request merged**